datafusion-examples/examples/composed_extension_codec.rs

```diff
@@ -233,6 +234,17 @@ impl PhysicalExtensionCodec for ChildPhysicalExtensionCodec {
     }
 }
 
+/// BlobFormatProto captures data encoded by blob format codecs
+#[derive(Clone, PartialEq, prost::Message)]
+struct BlobFormatProto {
+    /// encoder id used to encode blob
+    /// (to be used for decoding)
+    #[prost(uint32, tag = 1)]
+    pub encoder_position: u32,
+    #[prost(bytes, tag = 2)]
+    pub blob: Vec<u8>,
+}
```

Comment:
can we find a bit better name for this structure. Would it make sense to change doc string to something like: Structure captures information about encoder used and actual encoded data